.github/workflows/coverage.yml

```diff
@@ -0,0 +1,63 @@
+name: coverage
+
+on:
+  pull_request:
+  schedule:
+    # Prime the caches every Monday
+    - cron: 0 1 * * MON
```

Comment:
Shouldn't we only run it on the `main` branch as a reference? Or is it required to run the coverage test every Monday by `Codecov`?

Reply:
This comes straight from the example workflow for `setup-ocaml` [here](https://github.com/ocaml/setup-ocaml#example-workflow). I guess the idea is to check that the cache is populated each Monday, to avoid the first person who makes a PR each week needing to wait for an OCaml compiler to rebuild from source. Some explanation [here](https://github.com/ocaml/setup-ocaml/pull/63#discussion_r577735262).

Reply:
Oh OK! Thanks for the clarification :+1: